source/NoteView.cpp

```diff
@@ -602,6 +607,7 @@ void NoteView :: _AboutRequested(){
 	about->AddAuthors(authors);
 	about->AddText(B_TRANSLATE("Distributed under the terms of the GPLv2 license"));
 	about->AddText(B_TRANSLATE("Icons by Meanwhile"));
+	about->AddExtraInfo("extra info");
```

Comment:
Not sure why I don't find the text "extra info" displayed in the About window, but it shouldn't be in the cod ein the first place...

Reply:
Hello!
In the app there are two About windows ... One for the app and one for the replicant. It's redundand, and it's visible in the replicants one. Do you have a hint how I can achieve to use one About function only instead of two?